enterprise/server/quota/quota_manager.go

```diff
@@ -54,7 +62,38 @@ type QuotaManager struct {
 	bucketFactory bucketFactory
 }
 
+func (qm *QuotaManager) checkGroupBlocked(ctx context.Context) error {
+	fp := qm.env.GetExperimentFlagProvider()
+	if fp == nil {
+		return nil
+	}
+
+	if !fp.Boolean(ctx, disallowBlockedGroupsFlagKey, false) {
```

Comment:
should we make this a regular server flag rather than an experiment flag? (not sure we need the flexibility of an experiment here?)

Reply:
Im ok with either. The nice thing about this is we can disable immediately if something stops working. 

Since it's already guarded by the quota flag, we can potentially just remove this, and use that to control everything.